about.html

```diff
@@ -0,0 +1,162 @@
+                <li><a href="#">Documentation</a></li>
+                <li><a href="about.html">About</a></li>
+            </ul>
+    
+        </div>
+
+        <div class="navbar-desktop invisible"></div>
+    </nav>
+    <main class="main about-main">
+        <section class="headline">
+            <div class="headline-opacity">
+                <header class="headline-header">
+                    <h1>Africa's Talking Open Hackathon: FinTech</h1>
+                </header>
+                <div class="headline-desc">
+                    <p>
+                        Join us at our open hackathon every last Thursday of the month.
+                         Come build with us, present feature request, give feedback or 
+                         just contribute to our open source API's and SDKs. 
+                         Africa's Talking community allows developers to learn skills for the modern-day African Developer. 
+                         We are language and framework agnostic. All developers are welcome    
+                    </p>
+                </div>
+                <div class="when-where">
+                    <div class="when">
+                        <div class="when-header">
+                            <a class="fa far fa-calendar" href="#"></a>
+                            <h3> When</h3>
+                        </div>
+                        <div class="when-desc">
+                            <p>
+                                Thursday, Jun 30 2022<br>
+                                9:00 AM - 5:30 PM (EAT)
+                            </p>
+                        </div>
+                    </div>
+                    <div class="when">
+                        <div class="when-header">
+                            <a class="fa fa-map-marker" href="#"></a>
+                            <h3> Where</h3>
+                        </div>
+                        <div class="when-desc">
+                            <p>
+                                Africa's Talking HQ<br>
+                                Mzima Springs Rd, Lavington, Nairobi<br>
+                                Nairobi, 00100<br>
+                            </p>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </section>
+        <div class="about-desc">
+            <h3>Africa's Talking</h3>
+            <p>
+                powers communications solutions across Africa with simplified 
+                access to telco infrastructure, developers use our powerful SMS, 
+                USSD, Voice, Airtime, and Payments APIs to bring their ideas to life, 
+                as they build and sustain scalable businesses.
+            </p>
+        </div>
+        <section class="past-summit">
+            <div class="summit-header about-desc">
+                <h3>See The past Africa's Talking</h3>
+                <p>
+                    Tale a look at the last two Africa's talking 
```

Comment:
- [ ] This section should contain a divider after the title, kindly add it so your design is as close to the original design as possible

|Your design|CC design|
|-|-|
|![Screenshot_20220623_201302](https://user-images.githubusercontent.com/25227022/175379249-564d37bd-6098-4241-a07e-fd2633a17df3.png)|![Screenshot_20220623_201246](https://user-images.githubusercontent.com/25227022/175379277-bc81c02a-7f05-4d14-a695-7aab46bc8b0a.png)|